ERCS/erc-draft-embedding_registry.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Decentralized Embedding Registry  
+description: A standard to register embedding search service through a contract. 
+author: Crypti Talk <cryptitalk@cryptitalk.com> 
+discussions-to: TODO 
```

Comment:
Add discussions on Eth Magicians with a link to this PR.

Reply:
thanks done